app/scripts/svgpathdata.js

```diff
@@ -79,7 +79,13 @@ export class SvgPathData {
   transform(pointTransformer) {
     this.commands_.forEach(({command, args}) => {
       if (command == '__arc__') {
-        // TODO: transform arcs
+        // TODO: apply the transformation to the middle indicies as well.
```

Comment:
Sure thing.

BTW, I'm not missing anything here right? As in even though it still doesn't cover all cases... this additional code should still improve things, right?